src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -215,13 +215,14 @@ private String getSubmissionFormTableHeaderFragmentHtml(String questionNumberStr
         StringBuilder tableHeaderFragmentHtml = new StringBuilder();
         String tableHeaderFragmentTemplate = FeedbackQuestionFormTemplates.RUBRIC_SUBMISSION_FORM_HEADER_FRAGMENT;
 
-        for(int i = 0 ; i < numOfRubricChoices ; i++) {
+        for (int i = 0; i < numOfRubricChoices; i++) {
             String tableHeaderCell = 
                     FeedbackQuestionFormTemplates.populateTemplate(tableHeaderFragmentTemplate,
                             "${qnIndex}", questionNumberString,
                             "${respIndex}", responseNumberString,
                             "${col}", Integer.toString(i),
-                            "${rubricChoiceValue}", rubricChoices.get(i));
+                            "${rubricChoiceValue}", (Sanitizer.sanitizeForHtml(rubricChoices.get(i))
+                             + " (" + (numOfRubricChoices - i) + ")"));
```

Comment:
Two issues here:
- I don't think you need to wrap the entire `Sanitizer.sanitizeForHtml(...)` thing with brackets.
- Indent the 2nd line so that it falls directly below the `Sanitizer` part, something like:
```java
"${rubricChoiceValue}", Sanitizer.sanitizeForHtml(rubricChoices.get(i))
                        + " (" + (numOfRubricChoices - i) + ")");
```